app/views/tag_wranglings/index.html.erb

```diff
@@ -1,23 +1,88 @@
+      </li>
+      <li>
+        <%= t(".general_resources.current_wrangling_limitations_html",
+              current_wrangling_limitations_link: link_to(t(".general_resources.current_wrangling_limitations"), "https://wiki.transformativeworks.org/mediawiki/Current_Wrangling_Limitations")) %>
+      </li>
+      <li>
+        <%= t(".general_resources.tag_wrangling_faq_html",
+              tag_wrangling_faq_link: link_to(t(".general_resources.tag_wrangling_faq"), "https://wiki.transformativeworks.org/mediawiki/Tag_Wrangling_FAQ")) %>
+      </li>
+      <li>
+        <%= t(".general_resources.tag_wrangling_communication_policy_html",
+              tag_wrangling_communication_policy_link: link_to(t(".general_resources.tag_wrangling_communication_policy"), "https://wiki.transformativeworks.org/mediawiki/Wrangling_Communication_Policy")) %>
+      </li>
+      <li>
+        <%= t(".general_resources.tag_wranglers_mailing_list_html",
+              tag_wranglers_mailing_list_link: link_to(t(".general_resources.tag_wranglers_mailing_list"), "https://groups.google.com/a/transformativeworks.org/g/tagwranglers")) %>
+      </li>
+      <li>
+        <%= t(".general_resources.vacations_and_retiring_html",
+              vacations_and_retiring_link: link_to(t(".general_resources.vacations_and_retiring"), "https://wiki.transformativeworks.org/mediawiki/Vacations_and_Retiring_from_Wrangling")) %>
+      </li>
+      <li>
+        <%= t(".general_resources.tag_wrangling_bugs_html",
+              tag_wrangling_bugs_link: link_to(t(".general_resources.tag_wrangling_bugs"), "https://wiki.transformativeworks.org/mediawiki/Tag_Wrangling_Bugs_and_Feature_Wishlist")) %>
+      </li>
+      <li>
+        <%= t(".general_resources.otw_slack_chat_html",
+              otw_slack_chat_link: link_to(t(".general_resources.otw_slack_chat"), "https://transformativeworks.slack.com/"),
+              tag_wrangling_slack_tutorial_link: link_to(t(".general_resources.tag_wrangling_slack_tutorial"), "https://wiki.transformativeworks.org/mediawiki/Tag_Wrangling_Slack_Tutorial")) %>
+      </li>
+      <li>
+        <%= t(".general_resources.fandoms_in_need_html",
+              fandoms_in_need_link: link_to(t(".general_resources.fandoms_in_need"), "https://wiki.transformativeworks.org/mediawiki/Fandoms_in_need_of_Cowranglers")) %>
+      </li>
+      <li>
+        <%= t(".general_resources.unassigned_fandoms_html",
+              unassigned_fandoms_link: link_to(t(".general_resources.unassigned_fandoms"), unassigned_fandoms_path(sort: "count"))) %>
+      </li>
+      <li>
+        <%= t(".general_resources.ao3_catch_and_release_html",
+              ao3_catch_and_release_link: link_to(t(".general_resources.ao3_catch_and_release"), tag_wrangler_path("Ao3_Catch_and_Release")),
+              ao3_wrangling_faq_link: link_to(t(".general_resources.ao3_wrangling_faq"), "https://wiki.transformativeworks.org/mediawiki/AO3_Wrangling_Project_FAQ")) %>
```

Comment:
The "project" part of this link is important, so I'd prefer something like `wrangling_project_faq*` or `ao3_wrangling_project_faq*`